### PR TITLE
try handle non string message as hex 

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/iotagent-ul/issues/530

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add: allow handle binary messages
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: allow handle binary messages (#530)
```

File: lib/commonBindings.js [hunk 2]
```diff
@@ -203,14 +205,15 @@ function multipleMeasures(apiKey, device, message) {
  * @param {Object} device           Device object containing all the information about a device.
  * @param {Buffer} message          Raw message coming from the client.
```
Comment (fgalan) [inline lib/commonBindings.js:206]: Add `messageStr` to JavaDoc, explaining the difference with `message`

File: lib/commonBindings.js [hunk 3]
```diff
@@ -156,20 +156,22 @@ function processMeasureGroup(device, apikey, previous, current, index) {
...
-    const messageStr = message.toString();
     let parsedMessage;
 
     config.getLogger().debug('Processing multiple measures for device [%s] with apiKey [%s]', device.id, apiKey);
 
+    // Parse Message
     try {
         parsedMessage = ulParser.parse(messageStr);
     } catch (e) {
-        config.getLogger().error(context, 'MEASURES-003: Parse error parsing incoming message [%]', messageStr);
```
Comment (fgalan) [inline lib/commonBindings.js:169]: Given that now the parser error is not possible, MEASURE-003 is no longer used as error. It should be removed from documentation at docs/operations.md

Comment (AlvaroVega) [reply lib/commonBindings.js:169]: done in https://github.com/telefonicaid/iotagent-ul/pull/532/commits/b77ad41a6c5427919b3afbfb666a70841d33bca2

File: lib/commonBindings.js [hunk 4]
```diff
@@ -154,6 +154,7 @@ function processMeasureGroup(device, apikey, previous, current, index) {
  *
  * @param {String} apiKey           API Key corresponding to the Devices configuration.
  * @param {Object} device           Device object containing all the information about a device.
+ * @param {String} messageStr       String UL payload.
  * @param {String} message          UL payload.
```
Comment (fgalan) [inline lib/commonBindings.js:158]:
```suggestion
 * @param {String} messageStr       UL payload parsed to string.
 * @param {String} message          UL payload.
```

File: lib/commonBindings.js [hunk 5]
```diff
@@ -203,6 +204,7 @@ function multipleMeasures(apiKey, device, messageStr, message) {
  * @param {String} apiKey           API Key corresponding to the Devices configuration.
  * @param {String} attribute        Name of the attribute to update.
  * @param {Object} device           Device object containing all the information about a device.
+ * @param {Buffer} message          String Raw message coming from the client.
  * @param {Buffer} message          Raw message coming from the client.
```
Comment (fgalan) [inline lib/commonBindings.js:208]:
```suggestion
 * @param {Buffer} messageStr       Raw message coming from the client parsed to string.
 * @param {Buffer} message          Raw message coming from the client.
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**